app/controllers/api/groups_controller.rb

```diff
@@ -161,6 +161,86 @@ def update_marks
         HttpStatusHelper::ERROR_CODE['message']['200'] }, status: 200
     end
 
+    def add_extra_marks
```

Comment:
I should have been more specific in the last comment.
Please change this to `create_extra_mark`